public/js/reducers/chartsReducer.js

```diff
@@ -0,0 +1,97 @@
+const startVals = [{
```

Comment:
These are never displayed because the data is fetched from the api as soon as the app is loaded. I think it's safe to get rid of it and use null values in the initial state